README.md

```diff
@@ -1,16 +1,58 @@
 # 2023 Fall Learning Analytics Hackathon
-Register here: https://events.ctlt.ubc.ca/events/2023-fall-learning-analytics-hackathon/
+Welcome to the #hack-la-2023 github repository.
 
-# Welcome!
-Welcome to the #hack-la-2023 github repository. Please [fork this repo](https://docs.github.com/en/get-started/quickstart/fork-a-repo) and share your project back with us. In this repo you should find
+Please register for the hackathon if you haven't ready done so: https://events.ctlt.ubc.ca/events/2023-fall-learning-analytics-hackathon/
 
-* a data dictionary
-* a data sample (full data is available to registrants through the Canvas course)
-  
-* a final_product folder
-  * add your final product description and reflection 
-  * add your presentation files or a link to your presentation - suggested areas to cover in `slides.pptx` (use this as a template, a guide, or just inspiration - it is not a requirement to answer all questions)
+## 📚 What You'll Learn
+By completing this workshopathon, you'll learn to:
```

Comment:
should we change this to something like "additional resources / info"? / at least remove reference to workshopathon

Reply:
Oops, fixed. I also moved this section down.

Reply:
I'll merge once @craigdsthompson has a chance to review on Monday.